libvast/test/system/active_partition.cpp

```diff
@@ -253,9 +253,6 @@ TEST(delegate query to store with all possible ids in partition when query is to
       FAIL(err);
     });
   REQUIRE_EQUAL(last_query_contexts.size(), 1u);
-  vast::ids expected_ids;
```

Comment:
I think it is worth to also change the name of the test. I can't find how to add a suggestion to not changed code, but my proposal for the new name is:

**delegate query to the store**